packages/office-addin-manifest/test/test.ts

```diff
@@ -623,27 +624,56 @@ describe("Unit Tests", function() {
       });
     });
   });
+  describe("export.ts", function() {
+    describe("exportMetadataPackage()", function() {
+      it("export manifest", async function() {
+        this.timeout(6000);
+        const testFolder = path.resolve("./testExecution");
+        const manifestPath = path.normalize("test/manifests/manifest.json");
+        const assetsPath = path.normalize("test/assets");
+        const outputPath = path.normalize(`${testFolder}/testPackage.zip`);
+        const outputFile = await exportMetadataPackage(outputPath, manifestPath, assetsPath);
```

Comment:
I see now why this function returns, but I think just testing the file exists is fine.